components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/StandaloneJobOrchestratorTests.cs

```diff
@@ -124,19 +124,96 @@ public class StandaloneJobOrchestratorTests : TempFileProviderBase {
+                .Select(i => new OpcNodeModel {
+                    Id = $"nsu=http://microsoft.com/Opc/OpcPlc/;s=FastUInt{i}",
+                })
+                .ToList();
+
+            var endpoints = Enumerable.Range(0, numberOfEndpoints)
+                .Select(i => GenerateEndpoint(i, opcNodes, false))
+                .ToList();
+
+            await _standaloneJobOrchestrator.PublishNodesAsync(endpoints[0]).ConfigureAwait(false);
+
+            exceptionResponse = $"{{\"Message\":\"Response 404 Nodes not found\"," +
+                $"\"Description\":{{\"DataSetWriterId\":\"DataSetWriterId0\"," +
+                $"\"DataSetWriterGroup\":\"DataSetWriterGroup\"," +
+                $"\"DataSetPublishingInterval\":1000," +
+                $"\"EndpointUrl\":\"opc.tcp://opcplc:50000\"," +
+                $"\"UseSecurity\":false,\"OpcAuthenticationMode\":\"anonymous\"," +
+                $"\"OpcNodes\":[{{\"Id\":\"nsu=http://microsoft.com/Opc/OpcPlc/;s=SlowUInt0\"}}]}}}}";
+
+            var opcNodes1 = Enumerable.Range(0, numberOfEndpoints)
+                .Select(i => new OpcNodeModel {
+                    Id = $"nsu=http://microsoft.com/Opc/OpcPlc/;s=SlowUInt{i}",
+                })
+                .ToList();
+            var endpointsToDelete = Enumerable.Range(0, numberOfEndpoints)
+                .Select(i => GenerateEndpoint(i, opcNodes1, false))
+                .ToList();
+
+            // try to unpublish a not published nodes.
+            await FluentActions
+                .Invoking(async () => await _standaloneJobOrchestrator
+                    .UnpublishNodesAsync(endpointsToDelete[0])
+                    .ConfigureAwait(false))
+                .Should()
+                .ThrowAsync<MethodCallStatusException>()
+                .WithMessage(exceptionResponse)
+                .ConfigureAwait(false);
+
+            // description equal to a json string
+            exceptionModel = new MethodCallStatusExceptionModel {
+                Message = "Response 404 Nodes not found",
+                Description = $"{{\"DataSetWriterId\":\"DataSetWriterId0\"," +
+                    $"\"DataSetWriterGroup\":\"DataSetWriterGroup\"," +
+                    $"\"DataSetPublishingInterval\":1000,\"EndpointUrl\":\"opc.tcp://opcplc:50000\"," +
+                    $"\"UseSecurity\":false,\"OpcAuthenticationMode\":\"anonymous\"," +
+                    $"\"OpcNodes\":[{{\"Id\":\"nsu=http://microsoft.com/Opc/OpcPlc/;s=SlowUInt0\"}}]}}",
+            };
+            serializeExceptionModel = _newtonSoftJsonSerializer.SerializeToString(exceptionModel);
+            serializeExceptionModel.Should().BeEquivalentTo(exceptionResponse);
+        }
+
+            [Fact]
+        public async Task Test_PublishNodes_NullOrEmpty() {
+            InitStandaloneJobOrchestrator();
+
+            var exceptionResponse = $"{{\"Message\":\"Response 400 null request is provided\",\"Description\":{{}}}}";
```

Comment:
`exceptionResponse` object does not seem to be used in `Test_PublishNodes_NullOrEmpty`. Can we remove it ?